### PR TITLE
Desktop, mobile version and dynamic HTML with JavaScript

### PR DESCRIPTION
**Pull request by MuhammadUsmanAslam**
- Mobile version designed with HTML and CSS
- Desktop version designed using media query
- Speakers section created using dynamic HTML and JavaScript

Here is the Introduction Video: https://www.loom.com/share/d93d875569584cbca7908b67e152cb02
Here is the Live Link: https://muhammadusmanaslam.github.io/programmer-summit/index.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,8 +140,6 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
 If you like this project...
```
Comment (omarsalem7) [inline README.md]: - [x] Kindly consider giving appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: index.html [hunk 2]
```diff
@@ -1,13 +1,137 @@
...
+                <img src="./assets/program/workshop.png" alt="">
+                <h4>Workshop</h4>
+                <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quo temporibus minus sequi.</p>
+            </li>
+            <li class="program-item">
+                <img src="./assets/program/network.png" alt="">
+                <h4>CC Ignite</h4>
+                <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quo temporibus minus sequi.</p>
+            </li>
+        </ul>
```
Comment (omarsalem7) [inline index.html]: - [x] Kindly consider matching your design with Behance design by reducing the size of the image and font to match it

| Your design  | Behance design|
| ------------- | ------------- |
| image  | image |

File: about.html [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    
+    <section class="logo-selection">
+        <h3>Programmer Summit 2023 Logo</h3>
+        <hr>
+        <p>The logo of Programmer Summit 2023 was decided through logo competition from 7. July - 12. July</p>
+        <div class="logo-selection__logo">
+            <img src="./assets/logo-dark.png" alt="">
+        </div>
+    </section>
+    <hr class="hr-full">
```
Comment (omarsalem7) [inline about.html]: - [x] Kindly consider matching your design with Behance by choosing the correct font, padding, and margin

| Your design  | Behance design|
| ------------- | ------------- |
| image  | image |

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @MuhammadUsmanAslam 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by pinkmoon25 — APPROVED** [review]
Hi @MuhammadUsmanAslam , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters :heavy_check_mark: 
Github flow :heavy_check_mark: 
Good PR & Commit :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**